build_script.py

```diff
@@ -26,6 +26,14 @@ def run(command):
     subprocess.check_call(command, shell=True)
 
 
+def _mkdirp(path):
```

Comment:
It is admittedly kind of silly, since this is just a single script, so really *every* function is "internal" :stuck_out_tongue: